### PR TITLE
New "Profile subpages navigation" Addon

### PR DESCRIPTION
**Pull request by devxan**
Makes switching through different user profile subpages easy.

### Changes

Adds a row of links to all the user profile subpages.

### Reason for changes

To make getting to different user profile subpages faster.

### Tests

I tried this Addon with all my other standard addons _(101 gg bois)_ and it works as intended. 

### Screenshots

<details><summary>Click here to uncollapse dropdown</summary>
You spot a wild pokemon. Catch?
image
I don't actually know how pokemon work...
image
Listening to <a href="https://www.youtube.com/watch?v=8CYy9jNmpXM">Ping! 2</a> be like 😌 
image
Here's one without other addons 😲 
image
Thanks for listening to my TED Talk. 

I recently installed VSCode so I can churn addons out like butter. <details><summary>🧈👀</summary> Also if anyone knows how Javascript works, please tell me.</details>
</details>

### CODE REVIEW

**Comment by RedGuy12** [comment]
You need to make it a module...

File: addons/profile-subpages-navigation/addon.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "description": "Navigate through user profiles with navigation links on sub-pages.",
+  "tags": ["community", "profiles"],
+  "enabledByDefault": false,
+  "versionAdded": "1.25.0",
+  "dynamicDisable": true,
+  "dynamicEnable": true,
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/users/*/projects/"]
```
Comment (RedGuy12) [inline addons/profile-subpages-navigation/addon.json:12]: An array can have multiple elements

File: addons/profile-subpages-navigation/addon.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      "matches": ["https://scratch.mit.edu/users/*/followers/"]
+    }
+  ],
+  "credits": [
+    {
+      "name": "Xan",
+      "link": "https://scratch.mit.edu/users/-Xanimation-/"
+    },
+    {
+      "name": "Zro617",
```
Comment (RedGuy12) [inline addons/profile-subpages-navigation/addon.json:41]: I assume he made the original? If so, put him first. Also utilize credit.note

**Review by RedGuy12 — CHANGES_REQUESTED** [review]

File: addons/profile-subpages-navigation/addon.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        "https://scratch.mit.edu/users/*/followers/"
+      ]
+    }
+  ],
+  "credits": [
+    {
+      "name": "Xan",
+      "link": "https://scratch.mit.edu/users/-Xanimation-/",
+      "note": "Addon version"
+    },
```
Comment (lisa-wolfgang) [inline addons/profile-subpages-navigation/addon.json:27]: Don't forget the `id` property

**Review by mxmou — CHANGES_REQUESTED** [review]

File: addons/profile-subpages-navigation/addon.json (hunk 3 above)
Comment (devxan) [reply addons/profile-subpages-navigation/addon.json:28]: I have updated it to include the `id` property in the credits array: [5eb41c2fde4a833d8a47ad97993b60ac5c6ec8b5](https://github.com/ScratchAddons/ScratchAddons/commit/5eb41c2fde4a833d8a47ad97993b60ac5c6ec8b5)

**Comment by apple502j** [comment]
Marking as draft due to quality issues.

**Comment by WorldLanguages** [comment]
@devxan If you'll continue to work on this, please tell me and I'll reopen

**Pull request closed without merging**